### PR TITLE
Branch fix ug bug

### LINKED ISSUE #206: [PE-D][Tester D] Unclear email format in UG
It is unclear what the "valid format" for emails are from the UG. Perhaps it'll be clearer to give an example or specify the format of an email.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: wrjgold/ped#4

### PR DESCRIPTION
**Pull request by arnav12344**
Fixes #206

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ UniVerse is more than just a **desktop app for managing contacts**—it is a pla
 - Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
+- Note that name cannot include prefixes that are already part of our commands
```
Comment (verakohh) [inline docs/UserGuide.md:62]: add a fullstop at the end of the line

**Comment by arnav12344** [comment]
Fixes #218

**Comment by arnav12344** [comment]
Fixes #205

**Comment by arnav12344** [comment]
Fixes #204

File: docs/UserGuide.md [hunk 2]
```diff
@@ -101,11 +103,11 @@ add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS u/UNIVERSITY m/MAJOR b/BIRTHDATE [w/
 
 Parameters:
 
-- `n/NAME`: Full name of the contact.
+- `n/NAME`: Full name of the contact. These can't be tags that a.
```
Comment (verakohh) [inline docs/UserGuide.md:108]: what does this mean?

**Comment by arnav12344** [comment]
Fixes #220

**Comment by arnav12344** [comment]
Fixes #208

**Review by verakohh — COMMENTED** [review]
minor changes needed

File: docs/UserGuide.md [hunk 3]
```diff
@@ -59,12 +59,16 @@ UniVerse is more than just a **desktop app for managing contacts**—it is a pla
...
 
+- Note that name cannot include prefixes that are already part of our commands.
+
 - Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
 - Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
+- Work experience parameter `[w/WORK_EXPERIENCE]…​` can only be used one time. <br>
```
Comment (verakohh) [inline docs/UserGuide.md:70]: remove the '...' after [w/WORK_EXPERIENCE] as it means that there can be more than one work experience for a contact

**Comment by arnav12344** [comment]
Fixes #220

File: docs/UserGuide.md (hunk 3 above)
Comment (arnav12344) [reply docs/UserGuide.md]: Fixed

**Review by kuiktzecheng — APPROVED** [review]
lgtm!

**Pull request merged**